run.go

```diff
@@ -434,76 +350,61 @@ func (d *InMAP) checkOutputNames(o map[string]string) error {
 // 'coxHazard(Relative Risk, Population, Mortality Rate)' which calculates a
 // deaths estimate based on the relative risk associated with PM 2.5 changes
 // and the baseline number of deaths.
```

Comment:
Should we actually just delete these comments about the built-in functions since they are already explained in the io.go code?